### ISSUE TITLE
<strong> tag with extra whitespace is parsed incorrectly

### ISSUE THREAD

**Issue by sendmenas** [stormz/kramdown-prismic#10]
I have an issue when parsing from html to rich text.
If I pass such text `<p>Some text in <strong>strong </strong> style</p>`, rich text json is created correctly. But if there is a whitespace inside the tag `<p>Some text in <strong> strong</strong> style</p>`, then this part of the text gets lost.
```
[
  {
    type: 'paragraph',
    content: { text: 'Some text in  style', spans: [] }
  }
]
```
Works the same if the space is in the beginning or end.
Am I missing something or is this a bug?

**Comment by francois2metz** [comment]
Hi! I was able to reproduce it. This seems to be a bug. I'll try to fix it later.

**Comment by sendmenas** [comment]
> Hi! I was able to reproduce it. This seems to be a bug. I'll try to fix it later.

I have made a workaround with regex, so no rush from my side 😸

**Comment by francois2metz** [comment]
I have released the v0.3.8 that fix this issue.
